webserver/grpc/src/main/proto/reflection.proto

```diff
@@ -0,0 +1,147 @@
+// Copyright 2016 The gRPC Authors
```

Comment:
Maybe an odd question... Because this is the first time this file is appearing in the Helidon codebase, should the copyright reflect 2025 as well as 2016?

Same question for the `.proto` file in `test` below.